tests/data/data.json

```diff
@@ -1192,6 +1192,9 @@
     "description": "children patrons",
     "organisation": {
       "$ref": "https://bib.rero.ch/api/organisations/org1"
+    },
+    "limits": {
+      "unpaid_subscription": false
```

Comment:
Is it required? If yes this needs data migration steps.

Reply:
@jma I include a alembic script to migrate existing PatronTypes ;-)